test/Microsoft.AspNet.Mvc.TagHelpers.Test/CacheTagHelperTest.cs

```diff
@@ -320,6 +318,47 @@ public void GenerateKey_WithMultipleVaryByOptions_CreatesCombinedKey()
         }
 
         [Fact]
+        public async Task ProcessAsync_DoesNotUseCache_OnPassingFalseWhenEnabled()
+        {
+            // Arrange
+            var cache = new Mock<IMemoryCache>();
+            cache.Setup(c => c.Set(
+                /*key*/ It.IsAny<string>(),
+                /*link*/ It.IsAny<IEntryLink>(),
+                /*state*/ It.IsAny<object>(),
+                /*create*/ It.IsAny<Func<ICacheSetContext, object>>()))
```

Comment:
nit: mock setups in MVC tend to use trailing comments e.g. `It.IsAny<object>(),    // state`